project/MimaExcludes.scala

```diff
@@ -64,6 +64,10 @@ object MimaExcludes {
             // SPARK-6492 Fix deadlock in SparkContext.stop()
             ProblemFilters.exclude[MissingMethodProblem]("org.apache.spark.SparkContext.org$" +
                 "apache$spark$SparkContext$$SPARK_CONTEXT_CONSTRUCTOR_LOCK")
+		  ) ++ Seq(
```

Comment:
The indentation here looks a bit off.  Mind indenting this back a bit so that it lines up with the other closing braces (e.g. line 63)?